<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>20</width>
      <height>741</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_baslat">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(142, 236, 186);
color: black;
font: 32px;</string>
    </property>
    <property name="text">
     <string>Baslat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sure">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>120</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Muayene suresi :</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>39</y>
      <width>720</width>
      <height>611</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_image">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>11</y>
       <width>651</width>
       <height>531</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="exposure_time">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>180</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Exposure Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>240</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Frequency</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="check_frequency">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>240</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>270</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray Threshold</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="gray_thres">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>270</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="interval">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>300</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>300</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="inspection_time">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Master Camera</string>
    </property>
   </widget>
   <widget class="QComboBox" name="master_select">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Cam</string>
    </property>
   </widget>
   <widget class="QComboBox" name="filter_select">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="image_name">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sec">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>620</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 163, 72);
color: black;
font: 32px;</string>
    </property>
    <property name="text">
     <string>Se√ß</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ileri">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>620</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(246, 97, 81);
color: black;
font: 32px;</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_geri">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(246, 97, 81);
color: black;
font: 32px;</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
